content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -3227,10 +3227,9 @@ const result = await prisma.user.update({
 ```
 
 ## Filter conditions and operators
-
 <Admonition type="info">
 
-**Note:** The following examples show how to compare columns in separate tables.
+**Note:** The following examples show how to compare columns in the same table.
```

Comment:
```suggestion
```

Reply:
Tbh, I find this sentence quite confusing overall and don't see its value. The bullets are useful, but why the note? What should it say? 

If we don't know what it should say, let's just drop the sentence and keep the useful bullets to get this merged?

Reply:
I think it is supposed to qualify the bullets and the examples.

The examples all compare a field to a value `name` and `'Eleanor'` for example.
But since 4.3.0 it is also possible to compare fields to fields, so `name` to `lastname` for example.

So the `equals` examples for are really only one specific kind of example, there should possible als obe one that shows comparing the field to another field. But seems that is currently document under another headline, "Compare columns in the same table", that the first bullet links to.

(That headline should be "Compare fields in the same model" at least as the current words are wrong in Prisma context.)

Suggestion:
1. Fix the linked-to headline (and the words in the admonition here)
2. Add example for field to field comparison here as well
3. Then indeed remove the Note as it is not needed any more